Packs/AWS-EC2/Scripts/AwsEC2SyncAccounts/AwsEC2SyncAccounts.py

```diff
@@ -100,6 +100,8 @@ def update_ec2_instance(account_ids: list[str], ec2_instance_name: str) -> str:
         return f'Successfully updated ***{ec2_instance_name}*** with accounts:'
     except StopIteration:
         raise DemistoException(f'AWS - EC2 instance {ec2_instance_name!r} was not found or is not an AWS - EC2 instance.')
+    except (TypeError, KeyError) as e:
```

Comment:
explain?

Reply:
When there's no `Core REST API` instance it fails on `TypeError: string indices must be integers`